packages/cloud_controller_ng/pre_packaging

```diff
@@ -2,13 +2,20 @@ set -e -x
 
 cd ${BUILD_DIR}/cloud_controller_ng
 
+input_file="Gemfile"
+file_content=$(<"$input_file")
+modified_content=$(echo "$file_content" | sed '/^group :development do/,/^end/ d' | sed '/^group :test do/,/^end/ d')
+echo "$modified_content" > "$input_file"
+
 rm -rf ./vendor/cache/
 
 set +e
 for i in {1..3}; do
   bundle config --local specific_platform true
   bundle config --local path ./vendor/cache
   bundle config set cache_all true
+  bundle config set --local without 'development test'
+  bundle install --all-platforms --no-cache
```

Comment:
1. I think this command is not needed. When I run `bundle package` it also fetches all the Gems and modifies the `Gemfile.lock` (i.e. removes the development and test gems).
2. I believe this command was never executed successfully. When I run `bundle install --all-platforms`, I get the following error: `Unknown switches "--all-platforms"`
  The script does not abort due to the problematic error handling (`set +e`). To prevent such errors, could you please move `set +e` and `set -e` around `bundle package`?

```
set +e
bundle package --all-platforms --no-install
exit_code="$?"
set -e
```